.gitignore

```diff
@@ -1,13 +1,25 @@
 
-*.py[cod]
-pycache/
+
+*.py[co]
 .idea/
 tmp/*
 *.log.*
 *.log
 tms/media/*
 .idea/*
 
+tms/tms.db
+### Python template
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# Environments
+.env
+.venv
+env/
+venv/
+
```

Comment:
did you make .env folder inside the project ?

Reply:
ok i change it

Reply:
no need to make env folder in project.